test/LPStaking.test.js

```diff
@@ -50,9 +58,12 @@ contract("LPStaking", accounts => {
 
         it('user1 should be able to stake 50+50 tokens', async () => {
 
-            await lp_token.approve(pool1.address, web3.utils.toWei('100'), {from: testUser1});
+            await lp_token.approve(pool1.address, web3.utils.toWei('1000'), {from: testUser1});
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/tranche-jibrel/new-staking/pullRequest?prid=8459503)
```suggestion
            await lp_token.approve(pool1.address, web3.utils.toWei("1000"), {from: testUser1});
```